org-formation/040-budgets/cur.yaml

```diff
@@ -0,0 +1,42 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Monthly Cost and Usage Report'
+Parameters:
+  resourcePrefix:
+    Type: String
+Resources:
+  CostReportBucket:
+    Type: "AWS::S3::Bucket"
+    Properties:
+      BucketName: !Sub "${resourcePrefix}-cost-reports"
+      AccessControl: PublicRead
+  CostReportBucketPolicy:
+    Type: "AWS::S3::BucketPolicy"
+    Properties:
+      Bucket: !Ref CostReportBucket
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Action:
+              - 's3:GetBucketAcl'
+              - 's3:GetBucketPolicy'
+              - 's3:PutObject'
+            Effect: Allow
+            Resource: !Join
+              - ''
+              - - 'arn:aws:s3:::'
+                - !Ref CostReportBucket
+                - '/*'
+            Principal:
+              Service: "billingreports.amazonaws.com"
+  MonthlyCostReport:
+    Type: "AWS::CUR::ReportDefinition"
+    Properties:
+      ReportName: !Sub "${resourcePrefix}-monthly-costs"
+      ReportVersioning: "OVERWRITE_REPORT"
+      RefreshClosedReports: true
+      TimeUnit: "MONTHLY"
+      Compression: "Parquet"
+      Format: "Parquet"
+      S3Region: !Ref AWS::Region
+      S3Prefix: !Ref resourcePrefix
```

Comment:
would this not set report name to `sagebase-sagebase-monthly-costs`?

Reply:
No, the prefix is not added to the report name, it is added to the begging of the S3 path.

Reply:
hmm, are you sure?  the doc says.. `S3Prefix: The prefix that Amazon Web Services adds to the report name when Amazon Web Services delivers the report. Your prefix can't include spaces.`

Reply:
Oh, you're right. I thought it was pulling that part from `ReportName` but the two values were very similar for the manually created report.